hiss/customauth/tests/view_tests/resend.py

```diff
@@ -6,29 +6,50 @@
 from shared import test_case
 from user.models import User
 
+from django.utils.encoding import force_bytes, force_text
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from customauth.tokens import email_confirmation_generator
+
+
 URL_REGEX = r"(?P<url>https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*))"
 
 
 class ResendActivationEmailView(test_case.SharedTestCase):
+    def isValidLink(self, user, url):
```

Comment:
Please use `snake_case` for function definitions.

Also, please provide a docstring describing what this function does, and add type annotations to the function.
```suggestion
    def is_valid_link(self, user: User, url: str) -> bool:
```